src/ramses_tx/ramses.py

```diff
@@ -1339,29 +1339,29 @@
     },
 }
 
-# ventilation speed
+# ventilation speed description
 _31DA_FAN_INFO: dict[int, str] = {
-    0x00: "off",
-    0x01: "speed 1, low",  # aka low
-    0x02: "speed 2, medium",  # aka medium
-    0x03: "speed 3, high",  # aka high
-    0x04: "speed 4",
-    0x05: "speed 5",
-    0x06: "speed 6",
-    0x07: "speed 7",
-    0x08: "speed 8",
-    0x09: "speed 9",
-    0x0A: "speed 10",
-    0x0B: "speed 1 temporary override",  # timer
-    0x0C: "speed 2 temporary override",  # timer
-    0x0D: "speed 3 temporary override",  # timer/boost? (timer 1, 2, 3)
-    0x0E: "speed 4 temporary override",
-    0x0F: "speed 5 temporary override",
-    0x10: "speed 6 temporary override",
-    0x11: "speed 7 temporary override",
-    0x12: "speed 8 temporary override",
-    0x13: "speed 9 temporary override",
-    0x14: "speed 10 temporary override",
+    0x00: "0 (off)",
+    0x01: "1 (low)",
+    0x02: "2 (medium)",
+    0x03: "3 (high)",
+    0x04: "4",
+    0x05: "5",
+    0x06: "6",
+    0x07: "7",
+    0x08: "8",
+    0x09: "9",
+    0x0A: "10",
+    0x0B: "1 temporary override",  # timer
+    0x0C: "2 temporary override",  # timer
+    0x0D: "3 temporary override",  # timer/boost? (timer 1, 2, 3)
+    0x0E: "4 temporary override",
+    0x0F: "5 temporary override",
+    0x10: "6 temporary override",
+    0x11: "7 temporary override",
+    0x12: "8 temporary override",
+    0x13: "9 temporary override",
+    0x14: "10 temporary override",
```

Comment:
please justify changing these - the old values are the gold standard

Reply:
As in `22F4` please condider:
Passing along the word 'speed' in a non-English UI is not advised from i18n (internationalization). Can be added in UI by user if desired, harder to take it out. The label should make clear what a value means, so perhaps one would want to replace 'fan_info' in the UI with 'Speed' _in the UI_.

Reply:
Will follow suggestion to localize in ramses_cc, and leave API strings in lib as is